src/demo/native/story.js

```diff
@@ -46,6 +46,68 @@ storiesOf('Demos', module)
 
     const colors = select('color', ['red', 'blue', 'white', 'black'], 'black');
 
+    window.onCallbackInput = ev => {
+      const { value } = ev.target;
+
+      let filteredIcons = icons;
+      let filteredImages = images;
+
+      const lengthOfPrefix = 2;
+
+      const renderAreaIcons = document.querySelector('.icons');
+      const renderAreaImages = document.querySelector('.images');
+      const iconHeader = document.querySelector('.iconHeader');
+      const imageHeader = document.querySelector('.imageHeader');
+
+      filteredIcons = filteredIcons.filter(icon => {
+        const { length } = icon.path.split('.svg.js')[0];
+        const iconName = icon.path.substr(
+          lengthOfPrefix,
+          length - lengthOfPrefix
+        );
+        const foundSearchTerm = iconName.search(value.trim()) > -1;
+        return foundSearchTerm ? icon : '';
+      });
+
+      filteredImages = filteredImages.filter(image => {
+        const { length } = image.path.split('.svg.js')[0];
+        const iconName = image.path.substr(
+          lengthOfPrefix,
+          length - lengthOfPrefix
+        );
+        const foundSearchTerm = iconName.search(value.trim()) > -1;
+        return foundSearchTerm ? image : '';
+      });
+
+      renderAreaIcons.innerHTML = filteredIcons
```

Comment:
Can you use variables for SVG-names instead of change the whole innerHTML?

Reply:
I don't know exactly what you mean